.env.example

```diff
@@ -2,6 +2,7 @@
 HOSTNAME=127.0.0.1
 PORT=3000
 PROTOCOL=http
+API_PATH=/api/v1/
```

Comment:
Whats the reasoning behind this configurability? Tailor API in future should not be available for integrations. 🤔

Reply:
Here we just followed what we already have in boutique and primary reason is moving hardcoded value out of code: https://github.com/ExtensionEngine/tailor/blob/1abb956/client/api/request.js#L4

Reply:
Config still feels like adding it to the wrong place since it doesn't make sense for it to be configured. Or does it 🤔?

Reply:
Semantically speaking you are right which leads us to declaring it as global inside poi config?